src/polychron/presenters/ModelCreatePresenter.py

```diff
@@ -19,7 +19,12 @@ def __init__(self, mediator: Mediator, view: ModelCreateView, model: ProjectSele
         self.update_view()
 
     def update_view(self) -> None:
-        pass
+        """Update the view with the current project name."""
+        if self.model.next_project is not None:
+            project_name = self.model.next_project.name
```

Comment:
When a new project is being used, the `next_project` doesn't exist yet so is `None`, but `next_project_name` has the string name of the project that the user provided on the previous view.


Using `next_project` elsewhere is the better choice.

```suggestion
        if self.model.next_project_name is not None:
            project_name = self.model.next_project_name
```

It shouldn't ever show "New" under normal usage with this tweak, but having the fallback path is still useful (so it doesn't show `None`) instead.